instrumentation/opentelemetry-instrumentation-django/src/opentelemetry/instrumentation/django/middleware.py

```diff
@@ -88,6 +90,32 @@ def _get_span_name(request):
         except Resolver404:
             return "HTTP {}".format(request.method)
 
+    @staticmethod
+    def _get_metric_labels_from_attributes(attributes):
```

Comment:
I think this might be due to the fact that we are using the same `collect_attributes` method in wsgi for both client and server instrumentations. I think we probably should split this up in the future? For now I will follow the specs.